### PR TITLE
new site command

### PR DESCRIPTION
**Pull request by seakchiew**
Added scaffolding for webapp and rest application.
Changes on scaffolding on website.

### CODE REVIEW

File: commands/pixl8/new/site.cfc [hunk 1]
```diff
@@ -19,27 +19,82 @@ component {
...
+			appType = LCase(ask( "Enter site template: " ));
+		}
+		switch(LCase(appType)){
+
+		case "webapp":
+
+			var webDir = ListAppend( arguments.directory, "website", "/" );
+			
+			command( "preside new site" ).params(
+				skeleton = "preside-skeleton-webapp"
```
Comment (DominicWatson) [inline commands/pixl8/new/site.cfc]: We have a skeleton in Pixl8's private package repo. We should use that one.

File: commands/pixl8/new/site.cfc [hunk 2]
```diff
@@ -19,27 +19,82 @@ component {
...
+			).run();
+
+			break;
+
+		
+		case "rest":
```
Comment (DominicWatson) [inline commands/pixl8/new/site.cfc]: I'm not sure re the validity of this one. Could we leave out for now? If people want this they can just do `preside new site`

**Review by DominicWatson — CHANGES_REQUESTED** [review]
Looking good, couple of feedback things on the admin + rest skeletons (I think ditch rest skeleton for now + admin skeleton needs to use the internal Pixl8 one)

**Comment by seakchiew** [comment]
removed rest option and added pixl8 admin site skeleton

File: commands/pixl8/new/site.cfc (hunk 2 above)
Comment (seakchiew) [reply commands/pixl8/new/site.cfc]: removed

File: commands/pixl8/new/site.cfc (hunk 1 above)
Comment (seakchiew) [reply commands/pixl8/new/site.cfc]: changed to use pixl8 admin site skeleton. pending the skeleton to be packaged

**Comment by DominicWatson** [comment]
Nice one, all done and merged.

**Pull request merged**